containers/dev-local/docker-compose.yml

```diff
@@ -76,12 +201,29 @@ services:
       - SNAPSHOT_INTERVAL_SECONDS=1800
       - SNAPSHOT_OUTPUT_DIR=/tmp/snapshot-output
       - SNAPSHOT_NETWORK_ID=testnet
-      - DBSYNC_URL=postgres://user:password@host:port/dbname
+      - DBSYNC_URL=postgres://${DB_USER}:${DB_USER_PASSWORD}@db/${POSTGRES_DB}
       - GVC_API_URL=https://cardano.ideascale.com
     volumes:
       - leader0_store:/node_storage
       - leader0_store:/tmp/snapshot-output
 
+    # Send all logs to our internal fluentbit service.
+    logging:
+      driver: fluentd
+      options:
+        fluentd-address: ${FLUENT_BIT_HOST}:${FLUENT_BIT_PORT}
+
+  # Secondary Voting Node - 1 - TODO
```

Comment:
Perhaps we could consider implementing this type of TODO system. : `TODO(username): message, URL(GitHub or Jira issue)`

Reply:
We can probably introduce a custom clippy rule to automate it.

Reply:
Yes, having a uniform set of rules for TODO, etc would be awesome.  Especially if it can be linted with clippy.

Reply:
I [created an issue](https://github.com/input-output-hk/catalyst-core/issues/556) to implement such a rule. Please have a look.